src/OpAlgoBase.cpp

```diff
@@ -25,7 +25,7 @@ OpAlgoBase::OpAlgoBase(std::shared_ptr<vk::PhysicalDevice> physicalDevice,
         // If at least the x value is provided we use mainly the parameters
         // provided
         this->mKomputeWorkgroup = {
-            0,
+            komputeWorkgroup.x,
```

Comment:
Good catch, it would be worth adding a test in the C++ as well, as it hadn't been caught before